bin/oref0-bluetoothup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+if ! ( hciconfig -a | grep -q "PSCAN" ) ; then
+   sudo killall bluetoothd
+   sudo /usr/local/bin/bluetoothd --experimental &
+fi
+
+if ( hciconfig -a | grep -q "DOWN" ) ; then
+   sudo hciconfig hci0 up
+fi
+
+if !( hciconfig -a | grep -q $HOSTNAME ) ; then
```

Comment:
note the hostname is not actually needed. but it does help when you have multiple rigs paired to your phone, so you can tell what rigs are paired. it also helps with the Pebble, as I found I can have multiple rigs paired with one pebble (I haven't tried to see what happens when both rigs are running at same time)